tests/guardrails/max_bolus.tests.js

```diff
@@ -0,0 +1,62 @@
+const { limits } = require('../../src/index');
+
+var maxBolusTests = (test) => {
+    var screen;
+    it('open', async () => {
+        screen = await test.settingsScreen.OpenDeliveryLimitsScreen();
+    });
+    it('cannot set above the max limit', async () => {
+        await screen.Apply({
+            maxBolus: limits.bolusDelivery.max.limit + limits.bolusDelivery.step,
```

Comment:
Does this require a catch since we technically cannot set this value?

Reply:
yeah - detox fails as it can't find the value you have asked for